docs/api/read/programme.md

```diff
@@ -27,134 +75,182 @@ A programme is composed by a set of _offers_.
 ## Link Relations
    * A programme representation
       - must include a link to its context, using the self link relation.
-      - may include a link to the collection it belongs, using the collection link relation.
-      - must include a link to its _offers_.
-
-## Actions
-    * edit: edit's a programme
-    * add Offer: add's a curricular unit offer to the programme
+      - must include a link to the collection it belongs, using the `collection` relation.
+      - must include a link to the offers collection, using the `/rel/offers` relation.
 
-## Example representation
+## Example Representation
 ```json
 {
-    "class": [ "programme" ],
+    "class": ["programme"],
```

Comment:
This is a programme representation? If so, does it include all offers? It seems the program offer list should be a distinct resource.

Reply:
The list of offers is included in the programme representation so that it doesn't add breaking changes for now. The programme offers collection is also available at the same time, allowing the clients to update their behavior before changing the core representation.